.github/workflows/release.yaml

```diff
@@ -285,16 +298,26 @@ jobs:
   linux-build:
     strategy:
       matrix:
+        # Keep ubuntu-latest for general builds, add ubuntu-24.04-arm
+        os: [ ubuntu-latest ]
+        arch: [ amd64 ]
+        targets: [ 'archive', 'rocm' ] # Build both targets on amd64
         include:
+          - os: ubuntu-24.04-arm
+            arch: aarch64
+            targets: [ 'archive' ] # Only build archive target on aarch64
+          - os: ubuntu-22.04-arm
+            arch: aarch64
+            targets: [ 'archive' ]  # Only build archive target on aarch64
```

Comment:
These are necessary. The `linux-arm64` will produce artifacts for arm linux. Also, adding new branches here will produce artifacts that conflict with `linux-arm64` artifacts